libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/tests/PluggableColumnBarCharts.test.tsx

```diff
@@ -356,4 +368,45 @@ describe("PluggableColumnBarCharts", () => {
             });
         });
     });
+
+    describe("Drill Down", () => {
+        it.each([
+            [
+                insightDefinitionWithStackBy,
+                Department,
+                targetUri,
+                intersection,
+                expectedInsightDefinitionWithStackByDrillToDepartment,
+            ],
+            [
+                insightDefinitionWithStackBy,
+                Region,
+                targetUri,
+                intersection,
+                expectedInsightDefinitionWithStackByDrillToRegion,
+            ],
+            [insightDefinition, Region, targetUri, intersection, expectedInsightDefinitionDrillToRegion],
+        ])(
+            "should replace the drill down attribute and add intersection filters",
```

Comment:
The test name does not reflect the input, so if one fails, we would not know which.

Reply:
Fixed